benchopt/base.py

```diff
@@ -48,6 +50,21 @@ class BaseSolver(ParametrizedNameMixin, DependenciesMixin, ABC):
         strategy='iteration'
     )
 
+    @property
+    def solver_strategy(self):
+        """ Change stop_strategy to stopping_strategy """
+        if hasattr(self, 'stop_strategy'):
+            warnings.warn(
+                "'stop_strategy' attribute is deprecated \
+                    use 'stopping_strategy' instead",
+                DeprecationWarning
+            )
+            return getattr(self, 'stop_strategy')
+        elif hasattr(self, 'stopping_strategy'):
+            return getattr(self, 'stopping_strategy')
+        else:
+            return None
```

Comment:
The idea was to have the full mechanism to resolve the solver_strategy in this property:
```suggestion
            return self.stopping_criterion.strategy
```